actionRouter.js

```diff
@@ -0,0 +1,84 @@
+const express = require('express');
+
+const actionRouter = express.Router();
+
+const actionDB = require('./data/helpers/actionModel');
+
+actionRouter.get('/', (req,res) => {
+    actionDB.get()
+    .then( actions => {
+        res.status(200).json(actions);
+    })
+    .catch( err => {
+        status(500).json({error : 'server unale to retrieve actions'});
+    })    
+})
+
+actionRouter.get('/:id', (req,res) => {
+    const id = req.params.id;
+    actionDB.get(id)
+    .then( action => {
+        // if (action) {
+            res.status(200).json(action);
+        // } else {
+        //     res.status(404).json({error : `no user with id #: ${id}`});
+        // }    
+    })
+    .catch( err => {
+        res.status(500).json({error : 'server unable to retrive action'});
+    })
+})
+
+actionRouter.post('/', (req,res) => {
+    const newAction = req.body;
```

Comment:
you may want to do some data sanity checking here, ensuring that it is structured like you expect. you can destructure and only pass in the keys that you want to send, this way, if the client is trying to pass values that are not used/needed in this endpoint, you dont chance passing these into your db

Reply:
so what are some ways to do some data sanity checks?